frontend/src/scenes/data-warehouse/settings/source/Schemas.tsx

```diff
@@ -103,7 +103,7 @@ export const SchemaTable = ({ schemas, isLoading }: SchemaTableProps): JSX.Eleme
                                 </div>
                             </div>
                         ),
-                        tooltip: `Time of day in which the first sync will run. The sync frequency will be offset from the anchor time. This will not apply to sync intervals one hour or less.`,
+                        tooltip: `The sync frequency will be offset from the anchor time. This will not apply to sync intervals one hour or less.`,
```

Comment:
There is some additional code that prints the times that the syncs will occur. I think we'll want to update that to include the syncs that happen before the clock time.
<img width="447" alt="image" src="https://github.com/user-attachments/assets/d3f65e67-cb48-4c9a-8947-2f9251a51ee8" />

Reply:
Good point, was unaware of this 👍